File: Objetos_2_java/src/main/java/objetos2/java/gestionTareas/Tarea.java
package objetos2.java.gestionTareas;

public class Tarea {
    String nombre;
    String descripcion;
    Boolean completada;

    public Tarea(String nombre, String descripcion) {
        this.nombre = nombre;
        this.descripcion = descripcion;
        this.completada = false;
    }

    public void mostrarInfo(){
        System.out.println("Nombre: " + nombre + ", Descripcion: " + descripcion);
    }
}

File: Objetos_2_Kotlin/src/main/kotlin/gestorTareas/Tarea.kt
package gestorTareas

class Tarea(
    val nombre: String,
    private val descripcion: String,
    var completada: Boolean = false
) {
    override fun toString(): String {
        return "Nombre: $nombre, Descripcion: $descripcion, Completada: ${if (completada) "si" else "no"} )"
    }
}